packages/notifi-dapp-example/src/context/InjectiveWalletContext.tsx

```diff
@@ -0,0 +1,249 @@
+      setWalletKeysToLocalStorage('phantom', walletKeys);
+      setIsLoading(false);
+      return walletKeys;
+    } catch (e) {
+      throwError(
+        new Error('Leap connection failed, check console for details'),
+      );
+      console.error(e);
+    }
+    setIsLoading(false);
+    return null;
+  };
+
+  const leapSignArbitrary = useCallback(
+    async (message: string) => {
+      if (!walletKeysLeap || !injectiveLeapWallet.strategies.leap) {
+        return;
+      }
+      setIsLoading(true);
+      try {
+        const result = await injectiveLeapWallet.signArbitrary(
+          walletKeysLeap.bech32,
+          message,
+        );
+        return result;
+      } catch (e) {
+        throwError(
+          new Error('Leap signArbitrary failed, check console for details'),
+        );
+        console.error(e);
+      }
+      setIsLoading(false);
+    },
+    [walletKeysLeap],
+  );
+
+  const phantomSignArbitrary = useCallback(
+    async (message: string) => {
+      if (!walletKeysPhantom || !injectivePhantomWallet.strategies.phantom) {
+        return;
+      }
+      setIsLoading(true);
+      try {
+        const result = await injectivePhantomWallet.signArbitrary(
+          walletKeysPhantom.bech32,
+          message,
+        );
+        return result;
+      } catch (e) {
+        throwError(
+          new Error('Phantom signArbitrary failed, check console for details'),
+        );
+        console.error(e);
+      }
+      setIsLoading(false);
+    },
+    [walletKeysPhantom],
+  );
+
+  const disconnectLeapWallet = useCallback(async () => {
+    injectiveLeapWallet.disconnect;
+    selectWallet(null);
+    setWalletKeysLeap(null);
+    cleanWalletsInLocalStorage();
+  }, []);
+
+  const disconnectPhantomWallet = useCallback(async () => {
+    injectivePhantomWallet.disconnect;
+    selectWallet(null);
+    setWalletKeysLeap(null);
+    cleanWalletsInLocalStorage();
+  }, []);
+
+  const throwError = (e: Error, durationInMs?: number) => {
+    setError(e);
+    setTimeout(() => {
+      setError(null);
+    }, durationInMs ?? 5000);
+  };
+
+  const injectiveLeapWallet = new WalletStrategy({
+    chainId: ChainId.Mainnet,
+    wallet: Wallet.Leap,
+  });
+
+  const injectivePhantomWallet = new WalletStrategy({
+    chainId: ChainId.Mainnet,
+    ethereumOptions: {
+      ethereumChainId: EthereumChainId.Mainnet,
+      // rpcUrl: alchemyRpcEndpoint
+    },
+    wallet: Wallet.Phantom,
+  });
+
+  const wallets: InjectiveWallets = {
+    leap: new LeapWallet(
+      injectiveLeapWallet.strategies.leap ? true : false,
+      walletKeysLeap,
+      leapSignArbitrary,
+      connectLeapWallet,
+      disconnectLeapWallet,
+    ),
+    phantom: new PhantomWallet(
+      injectivePhantomWallet.strategies.phantom ? true : false,
+      walletKeysPhantom,
+      phantomSignArbitrary,
+      connectPhantomWallet,
+      disconnectPhantomWallet,
+    ),
+  };
+
+  useEffect(() => {
```

Comment:
for this feature(local storage so that users will be re-connected automaticly) you mentioned that we need to add, do you mean this section below?

Reply:
Ah .... right. But it seems not re-login.

https://github.com/notifi-network/notifi-sdk-ts/assets/127958634/2d0ad387-a2f0-4c33-893a-6ce98aa1e55f